docs/reference/tools-reference/index-lookup-tool.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Index lookup tool for flows in Azure AI Studio
+titleSuffix: Azure AI Studio
+description:  This article introduces the Index Lookup tool for flows in Azure AI Studio.
+author: e-straight
+manager: nitinme
+ms.service: azure-ai-studio
+ms.topic: conceptual
+ms.date: 01/18/2024
+ms.author: eur
+---
+
+# Index Lookup tool for Azure AI Studio
+
+[!INCLUDE [Azure AI Studio preview](../../includes/preview-ai-studio.md)]
+
+The prompt flow *Index Lookup* tool enables the usage of common vector indices (such as Azure AI Search, FAISS, and Pinecone) for retrieval augmented generation (RAG) in prompt flow. The tool automatically detects the indices in the workspace and allows the selection of the index to be used in the flow.
+
+## Build with the Index Lookup tool
+
+1. Create or open a flow in Azure AI Studio. For more information, see [Create a flow](../flow-develop.md).
```

Comment:
Since this repo also faces community local users. It would be better to have a specific guidance for local users. For example, local users need to install your tool package first. And they may don't know how to get the index url when setting the `mlindex_content` parameter.